crops/crop.go

```diff
@@ -121,33 +121,37 @@ func (c Crop) ComputeConsequences(event interface{}) consequences.Results {
 	return r
 }
 func (c Crop) computeImpactedCase(e hazards.ArrivalandDurationEvent) float64 {
-	// Determine damage percent based on damage dur curve and event dur
-	perdmg := c.lossFunction.ComputeDamagePercent(e)
-	fmt.Println("damage percent is ", perdmg)
-	croploss := (perdmg / 100) * c.GetValuePerOutputUnit()
-	fmt.Println("Crop loss is : ", croploss)
-	// value added to field before loss by production
-	fmt.Println("total pruduction costs are:", c.productionFunction.productionCostLessHarvest)
-	fmt.Println("fixed costs are: ", c.productionFunction.GetTotalFixedCosts())
-	loss := croploss + c.productionFunction.productionCostLessHarvest
-	fmt.Println("total loss is:", loss)
-	if loss > c.valuePerOutputUnit {
-		// Throw some error
-	}
-	return 10
+	// Determine crop damage percent based on damage dur curve and event dur
+	dmgfactor := c.lossFunction.ComputeDamagePercent(e) / 100
+	// Determin value added to field by production before loss
+	hazardMonth := e.ArrivalTime.Month() //iota "enum"
+	hazardMonthIndex := int(hazardMonth) - 1
+	loss := dmgfactor * (c.GetValuePerOutputUnit() + c.productionFunction.cumulativeMonthlyProductionCostsEarly[hazardMonthIndex])
```

Comment:
Good work. i think this is about right. i need to think through the math some more, but i think  you are on the right track.

Reply:
Reviewed and fixed with @HenryGeorgist